src/main/java/com/example/nlearn/models/User.java

```diff
@@ -0,0 +1,28 @@
+package com.example.nlern.models;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import jakarta.persistence.Id;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.ManyToOne;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "users")
+@Getter
+@Setter
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int id;
+
+    private String name;
+    private String email;
+    private String role;
```

Comment:
User should not have an arbitrary string for role, it shoulf be defined as enum and then mapped as a string using `@Enumerated` (refer to https://www.baeldung.com/jpa-persisting-enums-in-jpa)

Reply:
Done